tests/python/tir-transform/test_tir_transform_lower_cross_thread_reduction.py

```diff
@@ -116,7 +116,8 @@ def no_normal_reduction(a: T.handle, b: T.handle) -> None:
                 B[vi] = B[vi] + A[vi, vk]
 
 
-@T.prim_func
```

Comment:
Any idea what a fix would look like? I'd prefer not to have exceptions to the check without knowing what a well-formed version would look like

Reply:
In the `T.evaluate(T.tvm_thread_allreduce(...))` call, `k` should be replaced with `vk`.

Reply:
Thanks!

Reply:
Changing it causes the test to fail, so I think we need to change the transformation too.

Reply:
Yeah, I think this pass pre-dates schedulable TIR, and doesn't check for mapping of `ForNode::loop_var` to `BlockNode::iter_vars`.  For now, I'd probably just marked as ill-formed, to avoid expanding the scope of this PR.